exporters/stdout/stdoutlog/exporter_test.go

```diff
@@ -456,3 +462,493 @@ func TestValueMarshalJSON(t *testing.T) {
+					metricdatatest.IgnoreTimestamp(),
+					metricdatatest.IgnoreValue(),
+				)
+			},
+		},
+		{
+			name:   "duration",
+			enable: true,
+			test: func(t *testing.T, scopeMetrics func() metricdata.ScopeMetrics) {
+				exporter, err := New()
+				require.NoError(t, err)
+
+				rf := logtest.RecordFactory{
+					Timestamp: time.Now(),
+					Body:      log.StringValue("test log"),
+				}
+				record := rf.NewRecord()
+				records := []sdklog.Record{record}
+
+				err = exporter.Export(context.Background(), records)
+				require.NoError(t, err)
+
+				got := scopeMetrics()
+				assert.NotEmpty(t, got.Metrics)
+
+				assert.Equal(t, "go.opentelemetry.io/otel/exporters/stdout/stdoutlog", got.Scope.Name)
+				assert.NotEmpty(t, got.Scope.Version)
+
+				var durationMetric metricdata.Metrics
+				durationInstrument := otelconv.SDKExporterOperationDuration{}
+				for _, m := range got.Metrics {
+					if m.Name == durationInstrument.Name() {
+						durationMetric = m
+						break
+					}
+				}
+				require.NotEmpty(t, durationMetric, "duration metric not found")
+
+				expected := metricdata.Metrics{
+					Name:        durationInstrument.Name(),
+					Description: durationInstrument.Description(),
+					Unit:        durationInstrument.Unit(),
+					Data: metricdata.Histogram[float64]{
+						Temporality: metricdata.CumulativeTemporality,
+						DataPoints: []metricdata.HistogramDataPoint[float64]{
+							{
+								Count: 1,
+								Attributes: attribute.NewSet(
+									attribute.KeyValue{
+										Key:   "otel.component.name",
+										Value: attribute.StringValue(otelComponentType + "/0"),
+									},
+									attribute.KeyValue{
+										Key:   "otel.component.type",
+										Value: attribute.StringValue(otelComponentType),
+									},
+								),
+							},
+						},
+					},
+				}
+
+				metricdatatest.AssertEqual(
+					t,
+					expected,
+					durationMetric,
+					metricdatatest.IgnoreTimestamp(),
+					metricdatatest.IgnoreValue(),
+				)
+			},
+		},
+		{
+			name:   "multiple_exports",
+			enable: true,
+			test: func(t *testing.T, scopeMetrics func() metricdata.ScopeMetrics) {
+				exporter, err := New()
+				require.NoError(t, err)
+
+				for i := 0; i < 3; i++ {
+					rf := logtest.RecordFactory{
+						Timestamp: time.Now(),
+						Body:      log.StringValue("test log"),
+					}
+					record := rf.NewRecord()
+					records := []sdklog.Record{record}
+					err = exporter.Export(context.Background(), records)
+					require.NoError(t, err)
+				}
+
+				got := scopeMetrics()
+				assert.NotEmpty(t, got.Metrics)
+
+				assert.Equal(t, "go.opentelemetry.io/otel/exporters/stdout/stdoutlog", got.Scope.Name)
+				assert.NotEmpty(t, got.Scope.Version)
+
+				expectedAttrs := attribute.NewSet(
+					attribute.KeyValue{
+						Key:   "otel.component.name",
+						Value: attribute.StringValue(otelComponentType + "/0"),
+					},
+					attribute.KeyValue{Key: "otel.component.type", Value: attribute.StringValue(otelComponentType)},
```

Comment:
Please use the appropriate `semconv` package instead of defining these yourself.